engine/views/api_explorer.php

```diff
@@ -131,8 +131,8 @@ function clearContent(){document.getElementById("params").value="",setTimeout(fu
 
                         <p id="header-info"></p>
                         <p class="go-right">
-                            <input onclick="viewSettings()" class="button-default white-btn go-right" style="float-right; position: relative;" type="button" value="View Settings">
-                            <input onclick="copyText()" class="button-default white-btn go-right" style="float-right; position: relative;" type="button" value="Copy Response BODY">
+                            <input onclick="viewSettings()" class="button-default white-btn go-right" style="float:right; position: relative;" type="button" value="View Settings">
```

Comment:
```suggestion
                            <input onclick="viewSettings()" class="button-default white-btn go-right" type="button" value="View Settings">
```

Reply:
Has a small effect on the size of the modal. Without the inline styles the modal is a tad bigger. As these will probably be swapped out with the new admin theme, either way is fine I think.

Reply:
Also swaps buttons around.

Reply:
You were correct, the inline style "float:right;" will slightly affect the gap between those two buttons, that's why the modal was slightly different. **Moreover** see the screenshot blow, with the inline style "float:right;" it will change the order of the button as well.

### Without inline style
![截屏2020-12-12 下午11 40 08](https://user-images.githubusercontent.com/38635403/101984758-5febff80-3cd3-11eb-9d67-5d725618f9bb.png)

### With inline style
![截屏2020-12-12 下午11 39 37](https://user-images.githubusercontent.com/38635403/101984741-4d71c600-3cd3-11eb-951f-65ad2858ddb9.png)

Reply:
Yeah picked up on that - made changes to reflect.